src/bridges/example/ExampleBridge.sol

```diff
@@ -32,8 +32,8 @@ contract ExampleBridgeContract is BridgeBase {
         uint32[] memory gasUsage = new uint32[](2);
         uint32[] memory minGasPerMinute = new uint32[](2);
 
-        criterias[0] = computeCriteria(dai, dai);
-        criterias[1] = computeCriteria(usdc, usdc);
+        criterias[0] = uint256(keccak256(abi.encodePacked(dai, dai)));
+        criterias[1] = uint256(keccak256(abi.encodePacked(usdc, usdc)));
```

Comment:
I think it would be better if you called the `computeCriteria(...)` function here?